i18n/zh/docusaurus-plugin-content-docs/current/setup/install-with-keadm.md

```diff
@@ -32,30 +51,48 @@ KubeEdge 对 Kubernetes 的版本兼容性，更多详细信息您可以参考 [
 1. 必须正确配置 kubeconfig 或 master 中的至少一个，以便可以将其用于验证 k8s 集群的版本和其他信息。
 2. 请确保边缘节点可以使用云节点的本地 IP 连接云节点，或者需要使用 `--advertise-address` 标记指定云节点的公共 IP 。
 3. `--advertise-address`（仅从 1.3 版本开始可用）是云端公开的地址（将添加到 CloudCore 证书的 SAN 中），默认值为本地 IP。
-4. `keadm init` 将会使用二进制方式部署 cloudcore 为一个系统服务，如果您想实现容器化部署，可以参考 `keadm beta init` 。
+4. 在 v1.11.0 之后，`keadm init` 和 `keadm beta init` 默认实现容器化部署CloudCore，如果您希望以二进制包来部署CloudCore，请参考 [`keadm deprecated init`](#keadm-deprecated-init).
 
 举个例子：
 
 ```shell
-# keadm init --advertise-address="THE-EXPOSED-IP"(only work since 1.3 release)
+# keadm init --advertise-address="THE-EXPOSED-IP" --profile version=v1.16.0  --kube-config=/root/.kube/config (only work since 1.3 release)
```

Comment:
```suggestion
# keadm init --advertise-address="THE-EXPOSED-IP" --kubeedge-version=v1.17.0  --kube-config=/root/.kube/config (only work since 1.3 release)
```

`--kubeedge-version` is better than `profile version`, please modify this part also in English version

Reply:
changed according to the review comment